.github/workflows/unittest.yml

```diff
@@ -19,7 +19,6 @@ on:
       ktools_branch:
         description: "Build Ktools before tests: [git ref]"
         required: false
-        default: "develop"
```

Comment:
is this deletion on purpose?

Reply:
yup, i missed that from the PR notes, i'll add it in. 

I noticed when running a manual trigger, ktools is always built because that value is auto-filled with `develop` when it should probably be blank as default.